common/components/internal-header/template.njk

```diff
@@ -1,9 +1,9 @@
 <header class="app-header {{ params.classes if params.classes }}" role="banner" data-module="header"
         {%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
-  <div class="app-header__container {{ params.containerClasses | default('app-width-container') }}">
+  <div class="app-header__container {{ params.containerClasses if params.containerClasses }}">
```

Comment:
You can shorten this to:
```suggestion
  <div class="app-header__container {{ params.containerClasses }}">
```

If it has no value nunjucks will not output anything.